src/main/java/uk/gov/hmcts/reform/sscs/service/servicebus/TopicPublisher.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.reform.sscs.service.servicebus;
+
+import java.util.concurrent.atomic.AtomicReference;
+import javax.jms.ConnectionFactory;
+import javax.jms.JMSException;
+import javax.jms.Message;
+import javax.jms.Session;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.jms.IllegalStateException;
+import org.springframework.jms.connection.CachingConnectionFactory;
+import org.springframework.jms.core.JmsTemplate;
+import org.springframework.retry.annotation.Backoff;
+import org.springframework.retry.annotation.Recover;
+import org.springframework.retry.annotation.Retryable;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class TopicPublisher {
+
+    private final JmsTemplate jmsTemplate;
+
+    private final String destination;
+
+    private final ConnectionFactory connectionFactory;
+
+    @Autowired
+    public TopicPublisher(JmsTemplate jmsTemplate,
+                          @Value("${amqp.topic}") final String destination,
+                          ConnectionFactory connectionFactory) {
+        this.jmsTemplate = jmsTemplate;
+        this.destination = destination;
+        this.connectionFactory = connectionFactory;
+    }
+
+    @Retryable(
+        maxAttempts = 5,
+        backoff = @Backoff(delay = 2000, multiplier = 3)
+    )
+    public void sendMessage(final String message, String caseId, final AtomicReference<Message> msg) {
+        log.info("Sending message for caseId {}", caseId);
```

Comment:
could we please logs more specific to say this is from tribunals easy to debug ?